src/cdk/overlay/_overlay.scss

```diff
@@ -63,6 +63,10 @@ $backdrop-animation-timing-function: cubic-bezier(0.25, 0.8, 0.25, 1) !default;
     display: flex;
     max-width: 100%;
     max-height: 100%;
+
+    // Needs to be set specifically on the pane for
+    // the content to become scrollable on IE11.
+    overflow: auto;
```

Comment:
This is actually something I really wanted to avoid. The scrolling container has to live in the loaded component so that it can support virtual scrolling.

Reply:
In that case I'm not sure what the alternative would be. We may be able to have the scrolling container be `position: absolute; top: 0; bottom: 0;`, but it's hard to know which element is the scrollable one, without doing a bunch of measurements.